opm/simulators/utils/readDeck.hpp

```diff
@@ -67,6 +67,9 @@ prepareResultOutputDirectory(const std::string& baseName, const std::filesystem:
 
 std::unique_ptr<ParseContext> setupParseContext(const bool exitOnAllErrors);
 
+// Setup the OpmLog backends for screen output
+void setupStreamLogging(const std::string& stdout_log_id);
```

Comment:
I do not think `stdout_log_id` is necessary for this function and also setupLogging() function. but I might misunderstood something here.  Please suggest if there is some opinion here.